freespace_fetcher.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+from urllib import response
+from flask import Flask, g
+from sqlalchemy import insert, create_engine, text
+from sqlalchemy.orm import sessionmaker
+from models import db, Kapacita, OckovaciMisto, Dny
+from datetime import date
+import requests
+
+
+class FreespaceFetcher:
+    """
+    It is responsible for fetching data from the Reservatic plaftform.
+    API point: https://reservatic.com/public_services/411747/public_operations/5971/hours?date=2021-4-09
+    """
+    session = ''
+
+    def __init__(self):
+        some_engine = create_engine('postgresql://ockovani:ockovani2021@localhost:5432/ockovani')
+        Session = sessionmaker(bind=some_engine)
+        self.session = Session()
+
+    def fetch(self, service_id, operation_id, vacc_date):
+        """
+
+        @param service_id:
+        @param operation_id:
+        @param vacc_date: Date that we'd like to fetch
+        @return:
+        """
+        url = 'https://reservatic.com/public_services/{}/public_operations/{}/hours?date={}'.format(service_id,
+                                                                                                    operation_id,
+                                                                                                    vacc_date)
+        data = ''
+        print(url)
+        response_api = requests.get(url, data=data, headers={"Content-Type": "application/json"})
+
+        return response_api
+
+    def fetch_misto(self, misto_id, vacc_date):
+        """
+        It fetches necessary info from misto_id -> table ockovaci_misto and executes fetch for it.
+        Result will be stored in DB
+        @param misto_id: Misto that we'd like to fetch
+        @return:
+        """
+
+        misto_parameters = self.session.query(OckovaciMisto).from_statement(
+            text(
+                "SELECT * FROM ockovaci_misto WHERE misto_id=:misto_param"
+            )
+        ).params(misto_param=misto_id).all()
+
+        api_response = self.fetch(misto_parameters[0].service_id, misto_parameters[0].operation_id, vacc_date)
```

Comment:
Is it necessary to do the query? I would suggest to use two more arguments `service_id` and `operation_id` to the method `fetch_misto` and reuse already found values from `places_all`.